portfolio/src/main/webapp/index.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
-  <head>
+  
```

Comment:
Removing the `<head>` and `<body>` tags makes this a non-conforming HTML file. Browsers do their best with non-standard HTML but it's preferable to write conforming HTML because it parses faster and is more likely to look the same in different browsers.

Something like[HTML Tidy](https://www.html-tidy.org) can check your HTML for correctness.